packages/runtime/container-runtime/src/opLifecycle/opDecompressor.ts

```diff
@@ -76,9 +85,42 @@ export class OpDecompressor {
             const intoString = new TextDecoder().decode(decompressedMessage);
             const asObj = JSON.parse(intoString);
 
-            return { ...message, contents: asObj[0] };
+            return {
+                message: newMessage(message, asObj[0]),
+                state: "Processed",
```

Comment:
It's very hard to say looking at this code what these states impact, and what's the diff between them (for example processed vs. accepted from POV of the caller). Can they be more descriptive RE what is expected from the caller? Or what callee wants to communicate?

Reply:
https://github.com/microsoft/FluidFramework/pull/13329/files#diff-d0eaeb51fb9c18c0602e295b1e377df32547c432449f63a4824e691374c714abR49